modules/i2c/slavex.inc.c

```diff
@@ -6,7 +6,203 @@
  * The X_(p,s) macro must be defined before including.
  * It is automatically undefined at the end of this file.
  */
+#include <avr/interrupt.h>
+#include <stddef.h>
 
-#error Slave mode not supported yet
+#define I2CX(s) X_(I2C,s)
+#define i2cX(s) X_(i2c,s)
```

Comment:
You should define and use `i2csX(s) X_(i2cs,s)` instead, for method names and everything that is slave-specific.